src/main/java/io/github/moulberry/notenoughupdates/miscgui/minionhelper/Minion.java

```diff
@@ -107,7 +116,30 @@ public int getXpGain() {
 
 	@Override
 	public void onClick() {
-		NotEnoughUpdates.INSTANCE.manager.displayGuiItemRecipe(internalName);
+		if (Mouse.getEventButton() != 0 || !Keyboard.isKeyDown(Keyboard.KEY_LCONTROL)) {
+			NotEnoughUpdates.INSTANCE.manager.displayGuiItemRecipe(internalName);
+		} else {
+			if (minionSource instanceof CraftingSource) {
+				CraftingSource craftingSource = (CraftingSource) minionSource;
+				String bazaarName = null;
+				int totalAmount = 0;
+				for (Map.Entry<String, Integer> entry : craftingSource.getItems().entries()) {
+					String internalName = entry.getKey();
+					Integer amount = entry.getValue();
+					if (!APIManager.hardcodedVanillaItems.contains(internalName)) {
+						totalAmount += amount;
+						bazaarName = internalName;
+					}
+				}
+				MiscUtils.copyToClipboard(String.valueOf(totalAmount));
```

Comment:
Perhaps add a chat message for this? I also don't really get why this is useful

Reply:
Only a small addon for people who use skyhanni, so you can just paste it in signs.